Refresh.GameServer/Endpoints/AuthenticationEndpoints.cs

```diff
@@ -24,6 +24,24 @@ public LoginResponse Authenticate(RequestContext context, RealmDatabaseContext d
             ServerBrand = "Refresh",
         };
     }
+
+    private static readonly Lazy<string?> NetworkSettingsFile
+        = new Lazy<string?>(() => 
+        {
+            string path = Path.Combine(Environment.CurrentDirectory, "network_settings.nws");
+
+            return File.Exists(path) ? File.ReadAllText(path) : null;
+        });
+    
+    [GameEndpoint("network_settings.nws")]
+    public string? NetworkSettings(RequestContext context) {
+        string? networkSettings = NetworkSettingsFile.Value;
+        
+        if(networkSettings == null)
+            context.Logger.LogWarning(RefreshContext.Request, "network_settings.nws file is missing!");
```

Comment:
Hmm, I'd prefer this to be logged once; its possible the instance owner might not care as the game loads this file from disc if it cannot be pulled from the server.

Otherwise looks fine.